chain/client/src/chunk_validation.rs

```diff
@@ -81,8 +83,12 @@ impl ChunkValidator {
         &self,
         state_witness: ChunkStateWitness,
         chain_store: &ChainStore,
+        peer_id: PeerId,
     ) -> Result<(), Error> {
-        let chunk_header = state_witness.chunk_header.clone();
+        self.epoch_manager.verify_chunk_state_witness(&state_witness)?;
```

Comment:
Hmm, epoch_manager.verify_chunk_state_witness returns a Result<bool> but this code doesn't seem to care about the bool?

(Perhaps just return a Result<()> ?

also maybe call this verify_chunk_state_witness_signature. It's not actually verifying the witness.

Reply:
Oops, using the bool now.